script/ide/System/Database/test/test_TableDatabase.lua

```diff
@@ -641,4 +641,42 @@ function TestMultipleDB()
 	
 	db_cluster1.User:find({name="db",}, function(err, rows) assert(rows[1].email == "db_cluster1@1") end);
 	db_cluster2.User:find({name="db",}, function(err, rows) assert(rows[1].email == "db_cluster2@1") end);
-end
\ No newline at end of file
+end
+
+function TestOpenDatabase()
+	NPL.load("(gl)script/ide/commonlib.lua");
+	NPL.load("(gl)script/ide/LuaXML.lua");
+	NPL.load("(gl)script/ide/System/Database/TableDatabase.lua");
+	local TableDatabase = commonlib.gettable("System.Database.TableDatabase");
+
+	local config = { 
+		name = "tabledb", 
+		{
+			name = "providers", 
+			{ name = "provider", attr = { name = "raft", type = "TableDB.RaftSqliteStore", file = "(g1)npl_mod/TableDB/RaftSqliteStore.lua" }, "./, localhost, 9004, server4:" }
+		},
+		{
+			name = "tables",
+			{ name = "table", attr = { provider = "raft", name = "RaftUsers" } }, 
+			{ name = "table", attr = { provider = "raft", name = "RaftTemp" } },
+			{ name = "table", attr = { name = "DefaultToLocalProvider" } }, 
```

Comment:
[make default configurable](https://github.com/NPLPackages/main/pull/42/commits/dcc19f22476c2a0e0272a82fa28bae5c9fe9e889) implement the latter